bin/pcp-ping

```diff
@@ -4,6 +4,7 @@ require 'pcp/client'
 start_time = Time.now.to_f
 
 Thread.new { EM.run }
+Thread.pass until EM.reactor_running?
```

Comment:
I don't know enough about eventmachine to have a strong opinion. I just don't like the potential infinite loop here. If you are of the opinion it's virtually impossible then I'm OK with the PR.